pandas/types/concat.py

```diff
@@ -222,6 +222,9 @@ def union_categoricals(to_union, sort_categories=False):
     sort_categories : boolean, default False
         If true, resulting categories will be lexsorted, otherwise
         they will be ordered as they appear in the data.
+    ignore_order: boolean, default False
+        If true, ordered categories will be ignored.  Results in
```

Comment:
"ordered categories" -> "the ordered attribute of the categorical" / "whether the categorical is ordered or not" ? as the categories itself are not ignored, only its "orderedness"